client/webserver/site/src/js/app.ts

```diff
@@ -771,14 +793,15 @@ export default class Application {
     return () => { loader.remove() }
   }
 
-  /* orders retrieves a list of orders for the specified dex and market. */
+  /* orders retrieves a list of orders for the specified dex and market
+   * including inflight orders.
+   */
   orders (host: string, mktID: string): Order[] {
-    let o = this.user.exchanges[host].markets[mktID].orders
-    if (!o) {
-      o = []
-      this.user.exchanges[host].markets[mktID].orders = o
-    }
-    return o
+    let orders: Order[] = []
+    const mkt = this.user.exchanges[host].markets[mktID]
+    if (mkt.orders) orders = orders.concat(mkt.orders)
+    if (mkt.inflight) orders = orders.concat(mkt.inflight)
+    return orders
   }
 
```

Comment:
@buck54321, I think initializing `mkt.orders` or `mkt.inflight` with an empty array is not useful.